CHANGELOG.txt

```diff
@@ -30,3 +30,6 @@
   - Support checking statuses for Kubernetes resources.
 2.0.0.1:
   - Fix Conditional service check
+
+2.0.0.2:
+  - Add support for creating resources on Kubernetes using service account token
```

Comment:
Change version to 2.1.0. This is a new feature!

Reply:
Done

Reply:
2.1.0